pkg/api/org_users.go

```diff
@@ -124,33 +113,60 @@ func isOrgAdminFolderAdminOrTeamAdmin(c *models.ReqContext) (bool, error) {
 	return isAdminOfTeamsQuery.Result, nil
 }
 
+func (hs *HTTPServer) GetUsers(c *models.ReqContext) (models.SearchOrgUserQueryResult, error) {
```

Comment:
Nit, but I think it's better to simplify the return signature of the method to return the type required by the clients ([]*models.OrgUserDTO). It reduces complexity and the potential for mistakes.
```suggestion
func (hs *HTTPServer) GetUsers(c *models.ReqContext) ([]*models.OrgUserDTO, error) {
```